### PR TITLE
Page for users' guide on pedestal estimation tool 

### PR DESCRIPTION
**Pull request by tibaldo**
I wrote a first minimalistic version of a documentation page for the users' guide on the pedestal estimation tool. How does it look? Is there anything that should be added or changed?

### CODE REVIEW

**Comment by jlenain** [comment]
Hi @tibaldo!
Thanks a lot for this! The documentation builds nicely locally:
image
(I have to use another theme locally with respect to the ReadTheDocs site to compile properly).

It would be nice if you could (very) briefly describe the different filter methods (`WaveformsStdFilter` and `ChargeDistributionFilter`). Do you think it could be feasible?

File: docs/user-guide/howto-pedestal.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+To use the pedestal estimation tool, first activate the ``nectarchain`` environment:
+
+.. code-block:: console
+
+    $ mamba activate nectarchain
+
+
+The user script `nectarchain/user_scripts/ltibaldo/example_pedestal.py` showcases the usage of the tool.
+
+The input data are indentified by run number. See :ref:`env-vars` to set up the ``$NECTARCAMDATA`` environment variable. The pedestal estimation tool inherits the configurable parameters of the `~nectarchain.makers.component.PedestalComponent.PedestalEstimationComponent`.
```
Comment (jlenain) [inline docs/user-guide/howto-pedestal.rst:18]: Small typo: "indentified" -> identified

Comment (tibaldo) [reply docs/user-guide/howto-pedestal.rst]: Fixed.

**Comment by tibaldo** [comment]
Hi @jlenain, thanks for the review. I implemented the addition you suggested. I also added some information about the number of events per slice and the bad pixel flagging.

**Comment by jlenain** [comment]
Thank you very much, @tibaldo !

**Review by jlenain — APPROVED** [review]

**Pull request merged**